starling/src/starling/utils/RenderUtil.as

```diff
@@ -42,7 +42,10 @@ package starling.utils
                     Color.getRed(rgb)   / 255.0,
                     Color.getGreen(rgb) / 255.0,
                     Color.getBlue(rgb)  / 255.0,
-                    alpha);
+                    alpha,
+                    1.0, // depth
+					127  // stencil
```

Comment:
![tabsspacesboth](https://cloud.githubusercontent.com/assets/1126990/26819934/51a01baa-4aa1-11e7-907a-199e491fc9e4.png)
I'm however not against tabs in general. 😉